legacy/firmware/layout2.c

```diff
@@ -440,6 +442,15 @@ void layoutFeeOverThreshold(const CoinInfo *coin, uint64_t fee) {
                     _("Send anyway?"), NULL);
 }
 
+void layoutChangeCountOverThreshold(uint32_t change_count) {
+  char str_count[32] = {0};
```

Comment:
11 bytes should be enough for uint32

```suggestion
  char str_count[11] = {0};
```